contract/report/report_contract.xml

```diff
@@ -1,209 +1,192 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <odoo>
     <template id="report_contract_document">
-        <t t-call="web.html_container">
-            <t t-foreach="docs" t-as="o">
-                <t t-call="web.external_layout">
-                    <t t-set="o" t-value="o.with_context(lang=o.partner_id.lang)" />
-                    <t t-set="address">
-                        <p id="partner_info">
-                            <strong>Partner:</strong>
+        <t t-call="web.external_layout">
+            <t t-set="o" t-value="o.with_context(lang=lang)" />
```

Comment:
Is this still needed?

Reply:
Is necessary for translate partner information before starting the page with oe_structure

Reply:
Any update @pedrobaeza ?